popSimManu.tex

```diff
@@ -467,18 +471,19 @@ \subsection*{Use case: comparing methods of demographic inference}
 we show the inferred population size history (denoted $N(t)$)
 from samples taken from each of the three extant populations in the model.
 In each row we show comparisons among the methods (including two sample sizes for \MSMC).
-Blue lines show estimates from each of three replicate whole genome simulations and (FIXME: what color?)
-lines show the simulated population sizes.
+Blue lines show estimates from each of three replicate whole genome simulations. 
 When fitting single-population models to data simulated for multiple
 populations with migration, however, we do not necessarily expect the inference methods to recover the 
 ``true'' simulated population size due to model misspecification.
 Instead, many methods work by matching coalescence time distributions,
 and a single-population model with varying population size can match any coalescence time distribution
 (in which case coalescence rate is the inverse of the effective size).
 For this reason,
-we also plot the inverse coalescence rates calculated analytically in \texttt{msprime}
-as another benchmark of the ``truth'' in the single-population inference setting. 
-See the Appendix for a precise definition and description of the computation.
+we plot the inverse coalescence rates (black lines) calculated analytically in \texttt{msprime}
+as a benchmark of the ``truth'' in the single-population inference setting. Moreover, we show that 
+inverse coalescence rates closely approximate census population sizes, even for models with complex
+demographic histories (Figure \ref{fig:census_vs_IRC}).
```

Comment:
Maybe instead of the sentence starting "Moreover, ...", this should be:
> However, in tihs case inverse coalescence rates are very close to census population sizes (Figure \ref{fig:census_vs_IRC}).
... since we're only showing this for this one demographic history.

Reply:
sure, I think that is more appropriate in this context